fintech-examples/fintech-impl/pom.xml

```diff
@@ -149,6 +154,18 @@
             <version>${h2.database.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.esotericsoftware</groupId>
+            <artifactId>kryo</artifactId>
+            <version>5.0.0-RC1</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>de.adorsys.opba</groupId>
+            <artifactId>opba-banking-rest-api</artifactId>
+            <version>0.0.7-SNAPSHOT</version>
```

Comment:
Should be some property - `${project.version}`, also please check if this dependency is still needed

Reply:
Done